heroku/cli
# chore(apps-v5): add code coverage reports

## Why the change?
[GUS Card](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001NAwM0YAL/view)

As per our review of unit tests and coverage for the apps-v5 package, this PR does the following:
- Adds `nyc` package to add code coverage reports ✅
- Ensures that our code coverage reports over 85% test coverage of statements ✅
- Updated test name to align with testing purpose ✅

## Additional Notes
- Fixed the `heroku dashboard` command as we were never reporting the metrics info due to not resolving some promises
- `rename.js` and `destroy.js` are the only 2 at 71% and 76% Stmts coverage. Since these commands are trying to test `git` which isn't necessary for testing wrapped git commands, these files have been exempt from our >85% Stmts coverage acceptance criteria

## How to verify?
1. Pull down branch
2. Run tests
3. Confirm test coverage report is visible and that it reports over 85% test coverage of statements for all files except for `git.js`, `rename.js` and `destroy.js`
